browser/resources/settings/brave_help_tips_page/brave_help_tips_page.html

```diff
@@ -9,6 +9,10 @@
                           pref="{{prefs.brave.enable_window_closing_confirm}}"
                           label="$i18n{braveHelpTipsWarnBeforeClosingWindow}">
   </settings-toggle-button>
+  <settings-toggle-button class="cr-row"
```

Comment:
Can we put this on the system page actually? Great naming on the preference though - as it matches the Firefox setting name 😄

Reply:
done